transports/daily/CHANGELOG.md

```diff
@@ -5,7 +5,7 @@ All notable changes to **Pipecat Daily WebRTC Transport** will be documented in
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [Unreleased]
+## [0.3.5]
```

Comment:
Just looking and that is probably on me, but the 0.3.5 has already been released last month (`2025-02-13`). 

I guess with the features to support the buffering audio.

Reply:
https://www.npmjs.com/package/@pipecat-ai/daily-transport

Reply:
doh! good catch. fixed.